docs/logs/sensitive-pipeline/SensitiveLogging/IPiiLogger.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace SensitiveLogging;
```

Comment:
Is this tutorial focusing on "sensitive data processing" or "having more than one logging pipelines/destinations"? Who is the target audience?

Reply:
The scenario I got from @vishweshbankwar (based on a conversation with an azure monitor user):

> User wants to send certain logs (that may contain PII data) to endpoint 1 and rest to endpoint 2

I thought it would be a good thing to demo but I don't feel passionately we _must_ have it.

Reply:
I don't think we need to label it as "sensitive data processing" tutorial (it is specific use case that user had shared). 

I think the tutorial should focus on generalized thing i.e. how to set up more than one logging pipelines/destinations. It's not obvious to users how to set this up specially in asp.net core scenarios.

Reply:
> I think the tutorial should focus on generalized thing i.e. how to set up more than one logging pipelines/destinations. It's not obvious to users how to set this up specially in asp.net core scenarios.

I agree.

Reply:
is this something better suited at ILogger docs? This isn't tied to OTel in anyway, right

Reply:
Could be. But ILogger docs would not show the integration with OTel. So, having some example here would be helpful IMO.

Reply:
Updated to be just a general example for how to set up a dedicated pipeline